src/Microsoft.FeatureManagement.Telemetry.ApplicationInsights/TelemetryClientExtensions.cs

```diff
@@ -1,11 +1,10 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 //
-using Microsoft.ApplicationInsights;
 using Microsoft.ApplicationInsights.DataContracts;
 using Microsoft.FeatureManagement.FeatureFilters;
 
-namespace Microsoft.FeatureManagement.Telemetry.ApplicationInsights
+namespace Microsoft.ApplicationInsights
```

Comment:
We need to make the class name something less generic now that the namespace is generic.

Reply:
Otherwise it can cause conflicts if someone else does what we do.

Perhaps `FeatureManagementTelemetryClientExtensions` ?

Reply:
Agreed. Adjusted to `FeatureManagementTelemetryClientExtensions`